src/character_body.cpp

```diff
@@ -484,7 +485,8 @@ void Character::update_bodytemp()
             continue;
         }
 
-        // This adjusts the temperature scale to match the bodytemp scale,
+        // 0.01c is 5 body temp unit so it needs to times 5 to scale to match the bodytemp scale,
```

Comment:
This is super confusing,  I have no idea what this comments mean X)

Reply:
"need to be multiplied times 5" I guess. this is about the units not matching, and the problem was raised on the temperature class PR too